website/src/components/MultiDimensionalContentWidget.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import BrowserOnly from '@docusaurus/BrowserOnly';
+
+export const MultiDimensionalContentWidget = () => {
+
+	let getAllTabElements = function () {
+		var tabElements = document.querySelectorAll('.quickstart-tabs .tabs__item');
+		return tabElements;
+	}
+
+	let getByText = function (text) {
+		var allElements = getAllTabElements();
+		var selectedElement;
+		// docusaurus seems to be stripping away some array extensions like find()...
+		allElements.forEach(el => {
+			if (el.innerHTML == text) {
+				selectedElement = el;
+			}
+		})
+		return selectedElement;
+	}
+
+	let disableByText = function (text) {
+		var targetElement = getByText(text);
+		targetElement.classList.add('disabled-tab');
+	}
+
+	let enableByText = function (text) {
+		var targetElement = getByText(text);
+		targetElement.classList.remove('disabled-tab');
+	}
+
+	let selectByText = function (text) {
+		var targetElement = getByText(text);
+		targetElement.click();
+	}
+
+	let isSelectedByText = function (text) {
+		var targetElement = getByText(text);
+		var isSelected = targetElement.classList.contains('tabs__item--active');
+		return isSelected;
+	}
+
+	let jwtOnly = function () {
+		var isOnAuthPage = window.location.href.indexOf('/authentication') > -1;
+		return isOnAuthPage;
+	}
+
+	let bindTabs = function () {
+		setTimeout(function () {
+			var tabElements = getAllTabElements();
+			if (isSelectedByText('Besu') || isSelectedByText('Nethermind')) {
+				selectByText('HTTP-JWT');
+				disableByText('IPC');
+			}
+			if (isSelectedByText('IPC')) {
+				if (isSelectedByText('Besu') || isSelectedByText('Nethermind')) {
+					selectByText('Geth');
+				}
+				disableByText('Besu');
+				disableByText('Nethermind');
+			}
+			if (jwtOnly()) {
+				selectByText('HTTP-JWT');
+				disableByText('IPC');
+			}
+
+			tabElements.forEach(element => {
+				var isLabel = element.textContent.indexOf(":") > -1;
+
+				if (isLabel) {
+					// unbind so tab can't be selected
+					element.outerHTML = element.outerHTML;
+				} else {
+					element.addEventListener("click", function (event) {
+						var targetElement = event.target;
+						var textContent = targetElement.textContent;
+
+						if (textContent == 'Besu' || textContent == 'Nethermind') {
```

Comment:
this is a little janky 

 - if another developer `selected` the wrong item, they could enter an infinite loop
 - lots of magic strings 
 
IMO this is good enough for now barring any bugs, I can improve the devex and maintainability as part of a debt payoff PR